src/constants/app-constants.ts

```diff
@@ -302,6 +302,19 @@ export const TRACKER_TICKET_STATUS_MAP = {
     DONE: 'DoneStatus',
 };
 
+export const Retro_Grid_Service_Data = {
```

Comment:
Follow proper naming conventions and make it consistent with other parts of the application.

Reply:
changed